includes/class-amp-autoloader.php

```diff
@@ -71,7 +71,7 @@ class AMP_Autoloader {
 		'AMP_Analytics_Options_Submenu_Page'          => 'includes/options/views/class-amp-analytics-options-submenu-page',
 		'AMP_Options_Menu_Page'                       => 'includes/options/views/class-amp-options-menu-page',
 		'AMP_Rule_Spec'                               => 'includes/sanitizers/class-amp-rule-spec',
-		'AMP_Allowed_Tags_Generated'                  => 'includes/sanitizers/class-amp-allowed-tags-generated',
+//		'AMP_Allowed_Tags_Generated'                  => 'includes/sanitizers/class-amp-allowed-tags-generated',
```

Comment:
This file is being moved to the amp-library https://github.com/dennisinteractive/amp-library/tree/amphtml-generator/src/Spec